lib/Elastica/Client.php

```diff
@@ -62,19 +69,23 @@ class Client
      * @var LoggerInterface
      */
     protected $_logger = null;
+
     /**
-     * @var Connection\ConnectionPool
+     * @var BuilderInterface
      */
-    protected $_connectionPool = null;
```

Comment:
The connection pool remains protected, my habit of alphabetising properties strikes here, its just now on line 54.